.pre-commit-config.yaml

```diff
@@ -38,6 +38,7 @@ repos:
     rev: v0.7.1.1
     hooks:
       - id: shellcheck
+        files: 'scripts/'
```

Comment:
What about `docker/`?

Is this just because of `src/sentry/templates/sentry/reprocessing-script.sh` and if so, might just be better to specifically ignore that file on top of `types: [shell]`.

Reply:
Works for me.